docs/operations-guide/running-metabase-on-kubernetes.md

```diff
@@ -6,28 +6,121 @@ This guide will help you install Metabase on Kubernetes using [Metabase Helm cha
 
 * Kubernetes 1.4+ with Beta APIs enabled
 
-* [Kubernetes Helm](https://github.com/kubernetes/helm) installed
+* [Helm](https://github.com/helm/helm) installed
 
+### How to add Metabase official Helm repo
 
-### Installing
+```bash
+$ helm repo add metabase https://www.metabase.com/helm
+$ helm repo update
+```
+
+### How to run this chart
+```bash
+$ helm install metabase/metabase --generate-name
+```
+or, in the case you want to set a custom name:
+```bash
+$ helm install meta-helm-chart metabase/metabase
+```
 
-To install with the release name `my-release`:
+### Introduction
+
+This chart bootstraps a [Metabase](https://github.com/metabase/metabase) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager. It separates concerns about configs in ConfigMaps and Secrets and also sets up liveliness and readiness probes.
+
+This helm is highly configurable as it can be connected to any application database, tune Jetty or maximum data warehouse connections by passing configuration parameters (be aware that the default deployment will run with the H2 database which is [strongly discouraged](https://www.metabase.com/docs/latest/operations-guide/migrating-from-h2.html)).
+
+This helm pulls the `latest` docker tag of the `metabase/metabase` image, but it can be easily configured to pull Metabase Enterprise if needed as well as pinning down the version.
+
+### Uninstalling the Chart
+To uninstall/delete the `meta-helm-chart` deployment:
```

Comment:
This is assuming  `meta-helm-chart` was the custom name the person supplied, yeah?

Reply:
yup!